### PR TITLE
Mitaka for new installs

### PR DESCRIPTION
**Pull request by j2sol**
THis is a work in progress to get new installs on Mitaka code base.

Horizon is known to be hacked up to just function and needs work. Other services have deprecated config entries and whatnot that still need to be updated too, and magnum is just plain turned off.

This PR is just to get a view of the diffs, and won't pass CI yet due to networking issues.

### CODE REVIEW

File: roles/horizon/templates/etc/openstack-dashboard/local_settings.py [hunk 1]
```diff
@@ -1,97 +1,138 @@
...
-OPENSTACK_API_VERSIONS = {
-{% if horizon.keystone_api_version == 3 -%}
-    "identity": 3,
-{% else %}
-#     "identity": 3,
-{% endif %}
-    "volume": 2
-}
+# use of the decimal point, so valid options would be 2.0 or 3.
+#OPENSTACK_API_VERSIONS = {
```
Comment (zfeldstein) [inline roles/horizon/templates/etc/openstack-dashboard/local_settings.py]: this needs to be uncommented and we need identity == 3, in order for auth to work.

Comment (j2sol) [inline roles/horizon/templates/etc/openstack-dashboard/local_settings.py]: Fixed in a new push.

**Comment by j2sol** [comment]
retest

**Comment by nirajdp76** [comment]
LGTM .. some cleanup work can be done as we harden the Mitaka integration

**Pull request merged**